releaseNotes/SpringCloudAzureFeatureManagement.md

```diff
@@ -6,6 +6,33 @@
 
 [Source code web ][source_code_web] | [Package (Maven) web][package_web] | [Samples web][samples_web] | [Product documentation][docs]
 
+## 1.2.0 - January 13, 2020
+
+* Feature Management config files now match the DotNet format.
```

Comment:
Please use Java developer-friendly schema here and not mention "match the dotnet".